expression/builtin_encryption_vec.go

```diff
@@ -35,11 +35,54 @@ import (
 )
 
 func (b *builtinAesDecryptSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinAesDecryptSig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	strBuf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(strBuf)
+	if err := b.args[0].VecEvalString(b.ctx, input, strBuf); err != nil {
+		return err
+	}
+
+	keyBuf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(keyBuf)
+	if err := b.args[1].VecEvalString(b.ctx, input, keyBuf); err != nil {
+		return err
+	}
+
+	if b.modeName != "ecb" {
+		return errors.Errorf("unsupported block encryption mode - %v", b.modeName)
+	}
+
+	isWarning := !b.ivRequired && len(b.args) == 3
+
+	result.ReserveString(n)
+	for i := 0; i < n; i++ {
+		if strBuf.IsNull(i) || keyBuf.IsNull(i) {
+			result.AppendNull()
+			continue
+		}
+		if isWarning {
+			b.ctx.GetSessionVars().StmtCtx.AppendWarning(errWarnOptionIgnored.GenWithStackByArgs("IV"))
+		}
+		key := encrypt.DeriveKeyMySQL([]byte(keyBuf.GetString(i)), b.keySize)
+		plainText, err := encrypt.AESDecryptWithECB([]byte(strBuf.GetString(i)), key)
```

Comment:
```suggestion
		plainText, err := encrypt.AESDecryptWithECB(strBuf.GetBytes(i), key)
```

Reply:
we can't use `GetBytes` here. The reason is at https://github.com/pingcap/tidb/pull/13219#issuecomment-550973539

Reply:
Could you add an annotation here? And do the conversation at a seperated line.
```go
// annotation
str := []byte(strBuf.GetString(i))
plainText, err := encrypt.AESDecryptWithECB(str, key)
```